pyfr/solvers/euler/kernels/rsolvers/hllc.mako

```diff
@@ -1,60 +1,51 @@
 <%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
 <%include file='pyfr.solvers.euler.kernels.flux'/>
 
-<%pyfr:macro name='rsolve' params='ul, ur, n, nf'>
+<%pyfr:macro name='rsolve_1d' params='ul, ur, nf'>
```

Comment:
Can you check if this kernel previously vectorized for AVX-512/NEON that it still does so?

Reply:
Just following up here.

Reply:
Using godbolt with gcc 13.2 x86 and adding `-mavx2` (as there isn't access to `-march=native`) in addition to the standard pyfr flags, the HLLC kernel is vectorising. iei using `ymm` registers and using commands `vmulpd` etc.